app/templates/serviceLearning/slcQuestionnaire.html

```diff
@@ -7,33 +7,39 @@ <h2 class="text-center pt-4">New Service-Learning Course</h2>
   <label><b>
     1. Indicate the specific course learning objective(s) that will be addressed by the service-learning component.
   </b></label>
-  <textarea id="slcQuestionOne" class="form-control" aria-label="With textarea" name="1">{{questionanswers[0]|urlize if isViewSlc else questionanswers[0] if questionanswers}}</textarea>
+  <textarea id="slcQuestionOne" class="form-control textarea" aria-label="With textarea" name="1">{{questionanswers[0]|urlize if isViewSlc else questionanswers[0] if questionanswers}}</textarea>
+  <p class="charCounter">Characters: <span class="charCount">0</span></p>
```

Comment:
I think we should use more descriptive names than charCounter and charCount. Maybe something like slcQestionCharCounter and slcQuestionCharCount.